# Serie 3

## Aufgabe 3.1
Die monatlichen Aufwendungen X [CHF] für den Wasserverbrauch eines 2-Personenhaushalts seien durch eine Zufallsvariable mit der folgenden Wahrscheinlichkeitsdichtefunktion gegeben:

$$f(x)=\left\{
    \begin{matrix}
    cx(15-\frac{x}{4})\ && \text{falls}\ 0\leq x\leq 60 \\
    0 && \text{sonst}
    \end{matrix}
\right. $$

### a)

Welchen Wert muss $c$ annehmen?

$$\int\limits_0^{60}{cx(15-\frac{x}{4})} = 1 $$

$$\left.\frac{15\cdot c\cdot x^2}{2}-\frac{c\cdot x^3}{12}\right|_0^{60} = 1 $$

$$\frac{15\cdot c\cdot 60^2}{2}-\frac{c\cdot 60^3}{12} = 1 $$

$$\frac{54000\cdot c}{2} - 18000\cdot c = 1 $$

$$27000\cdot c - 18000\cdot c = 1$$

$$c = \frac{1}{9000}$$

In [7]:
print(str(15*60**2))
print(str(60**3/12))

54000
18000.0


### b)

Geben Sie die Verteilungsfunktion $F$ der Zufallsvariablen $X$ an.

$$ F(x)=\left\{
    \begin{matrix}
    0 && \text{falls}\ x<0\\
    \frac{1}{9000}(\frac{15}{2}x^2-\frac{1}{12}x^3)\ && \text{falls}\ 0\leq x\leq 60\\
    1 && \text{falls}\ x>60
    \end{matrix}
\right. $$

### c)

Welcher Wert der Aufwendungen wird nur mit 10% Wahrscheinlichkeit überschritten?

Gesucht ist ein $a$ sodass $P(X>a)=0.1$ also ist $P(X\leq a)=0.9$, somit muss gelten $F(a)=0.9$. Mit der Formel aus b):

$$\frac{1}{9000}(\frac{15}{2}a^2-\frac{1}{12}a^3) = 0.9$$

$$\frac{15}{2}a^2-\frac{1}{12}a^3 = 0.9\cdot 9000$$

$$\frac{15}{2}a^2-\frac{1}{12}a^3 - 8100 = 0\cdot 9000$$

In [10]:
import sympy as sym

a = sym.Symbol('a')
sym.solveset(15/2*a**2-1/12*a**3-8100)

{-28.6249720273366, 48.2519936604545, 70.3729783668821}

Davon liegt nur $a = 48.252$ im Wertebereich von $F(x)$ und ist somit unsere gesuchte Grenze.

### d)

Wie hoch sind die erwarteten monatlichen Aufwendungen für den Wasserverbrauch eines 2-Personenhaushalts?

Der Erwartungswert für stetige Zufallsvariablen $X$ ist wie folgt definiert:

$$\int\limits_{-\infty}^{\infty}{x \cdot f(x)\ dx} $$

Da unsere Funktion $f(x)$ aber nur im Intervall $[0,60]$ definiert ist, erhalten wir folgendes Integral:

$$\int\limits_0^{60}{x\cdot \frac{1}{9000}x\ (15-\frac{x}{4})}$$

$$= \frac{1}{9000}\int\limits_0^{60}{x^2\ (15-\frac{x}{4})}$$

$$= \frac{1}{9000}(\left. 5x^3-\frac{1}{16}x^4\right|_0^{60})$$

$$= 30$$

In [1]:
5*60**3

1080000

In [2]:
1/16*60**4

810000.0

In [4]:
1/9000*(1080000-810000)

30.0

## Aufgabe 3.2

In derStadt Luzern gibt es bekanntlich viele Baustellen. Die Dauer $X$ der Arbeiten bei einer Baustelle liege zwischen 0 und 20 Wochen. Die Dichte $f(x)$ habe die folgende Form:

![Dauer Arbeiten auf Baustellen in Luzern](img/ArbeitenBaustelleLuzern.png)

### a)
Begründen Sie, warum $c = 0.1$ ist und schreiben Sie die Dichte $f(x)$ explizit auf.

Da die Fläche unter der Wahrscheinlichkeitsdichtefunktion 1 ergeben muss.

Wenn wir das Ganze als Dreiecksfläche anschauen erhalten wir $\frac{c\cdot20}{2}=1$. und somit $c=\frac{1}{10}$. Die Wahrscheinlichkeitsdichtefunktion ist also:

$$f(x)=\left\{
    \begin{matrix}
    0\ && x<0\ \text{und}\ x>20 \\
    -\frac{1}{200}x+\frac{1}{10} && 0\leq x\leq 20
    \end{matrix}
\right. $$

### b)
Berechnen Sie die Wahrscheinlichkeit, dass die Bauzeit $X$ weniger als
1. 5 Wochen
2. 10 Wochen

beträgt.

$$g(t) = F(t) = \int\limits_{0}^{t}{-\frac{1}{200}x+\frac{1}{10}\ dx} = -\frac{1}{400}x^2+\frac{1}{10}x $$

Und somit:

$$g(5) = -\frac{5^2}{400}+\frac{5}{10}$$

$$g(10) = -\frac{10^2}{400}+\frac{10}{10}$$

In [5]:
-(5**2)/400+5/10

0.4375

In [6]:
-10**2/400+1

0.75

### c)

Skizzieren Sie die kumulative Verteilungsfunktion.

### d)

Berechnen Sie den Erwartungswert, den Median und die Standardabweichung der Dauer $X$.

### e)

$K = 40′000\cdot\sqrt{X}$ entspreche dem Betrag in Franken, den die Arbeiten bei einer Baustelle kosten. Wie gross ist die Wahrscheinlichkeit, dass die Arbeiten bei einer Baustelle höchstens 120’000.- Fr. kosten?

Die vorgeschlagene Verteilung ist nur ein Modell. Man könnte die Dauer der Bauarbeiten zum Beispiel auch als exponentialverteilt annehmen.

### f)
 Für welchen Parameter $\lambda$ hat die Exponentialverteilung denselben Erwartungswert wie die bisherige angenommene Verteilung?

 ### g)
Berechnen Sie mit der gefundenen Exponentialverteilung nochmals Teilaufgabe e).